.pre-commit-config.yaml

```diff
@@ -47,6 +56,15 @@ repos:
         files: ^mobile/
         types_or: [javascript, jsx, ts, tsx, css, markdown, json] # uses https://github.com/pre-commit/identify
         entry: bash -c 'cd mobile && npm run format'
+      - id: lintern-mobile
+        name: lintern-mobile
+        stages:
+          - commit
+          - merge-commit
+        language: system
+        files: ^mobile/
+        types_or: [javascript, jsx, ts, tsx, css, markdown, json] # uses https://github.com/pre-commit/identify
+        entry: bash -c 'cd frontend && npm run lint'
```

Comment:
We were not running frontend linterns after-commit and I always forgot about them